arch/arm/neon_intrins.h

```diff
@@ -1,8 +1,14 @@
 #ifndef ARM_NEON_INTRINS_H
 #define ARM_NEON_INTRINS_H
 
-#if defined(_M_ARM64) || defined(_M_ARM64EC)
-#  include <arm64_neon.h>
+#if (defined(_M_ARM64) || defined(_M_ARM64EC))
```

Comment:
These includes are also part of tests in CMake and configure build scripts.

I'm thinking it would be better to use a `check_include_file`, but @mtl1979 might have ideas on that.

Reply:
Actually it is only used in source in one place. So maybe it is not needed. But those other instances should be updated. But honestly it kind of seems to me like a toolchain bug.

Reply:
The problem with check_include_file() is that it might find a header for different target when toolchain supports multiple architectures as is with Windows API.